BrowserKit/Sources/Common/Utilities/FontStyle.swift

```diff
@@ -0,0 +1,56 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+
+public struct Styling {
```

Comment:
`Styling` is too general in my 👀 . I would be more specific, like `TextStyling` or something.

Reply:
Updated to `TextStyling` since I'm not sure if `TextStyle` will be confusing with the `UIFont.TextStyle` type. Will ask the team for their preference tomorrow too!